Fix `WebAssembly.Tag()` exception case wording (#17996)
mdn/content: files/en-us/web/javascript/reference/global_objects/webassembly/tag/tag/index.md

### Exceptions

- {{jsxref("TypeError")}}:
  - : Thrown if at least one of these conditions are met:

>   - : The `type` parameter is not an object.
>   - : The `type.parameters` property is not supplied.
>   - : The `type.parameters` contains an unsupported data type.

Reviewer: We want a `<ul>` inside a `<dt>`. This way:
```suggestion
    - The `type` parameter is not an object.
    - The `type.parameters` property is not supplied.
    - The `type.parameters` contains an unsupported data type.
```

### Exceptions

> - A {{jsxref("TypeError")}} will be thrown if `type` is not an object, the `type.parameters` property is not supplied, or `type.parameters` contains an unsupported data type.

Reviewer 2: What about when `type.parameters` is an empty array?

> - A {{jsxref("TypeError")}} will be thrown if `type` is not an object, the `type.parameters` property is not supplied, or `type.parameters` contains an unsupported data type.

Reviewer: We write exception sections as definition lists, they are much easier to parse and read by developers than whole sentences. (The content stays the same)

```suggestion
- {{jsxref("TypeError")}} 
  - : Thrown if `type` is not an object, the `type.parameters` property is not supplied, or `type.parameters` contains an unsupported data type.
```

Reviewer: I have no strong feeling about it here as these are short conditions. But this works too:
```suggestion
- {{jsxref("TypeError")}} 
  - : Thrown if one or several of the following condition is met:
    - The `type` parameter is not an object.
    - The `type.parameters` property is not supplied.
    - The `type.parameters` property contains an unsupported data type.
```
